packages/web-components/.storybook/preview-head.html

```diff
@@ -13,7 +13,8 @@
 
   .sbdocs-preview,
   div.docs-story {
-    background: var(--fill-color) !important;
+    background: var(--colorNeutralBackground2) !important;
+    color: var(--colorNeutralForeground2) !important;
```

Comment:
Did you mean to set color here?

Reply:
Yes, the components currently expect the color to be set according to theme.

Reply:
Yup makes sense. I'm not sure I'd set important though, This might make us miss styling bugs within a component

Reply:
that's fair, important removed